library/ui-styles/src/debug/res/layout/debug_social_login.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
debug layout to have a preview of all the buttons style

Reply:
is there a way to access this file? maybe a separate gallery app could be helpful (especially if we start adding more components to this module)

Reply:
There is already some debug screen that you can access by clicking the green setting icon from the drawer. For this layout I think a preview in AS is enough.